desktop/apps/categories2/components/GeneFamilies.js

```diff
@@ -1,23 +1,84 @@
 import React from 'react'
 
-const GeneFamily = (props) => {
+function partitionArray(array) {
```

Comment:
I'm not super happy about this function. But I also don't see the point in refactoring it to accept _N_ columns. Open to suggestions here!